src/ape/managers/config.py

```diff
@@ -53,7 +53,7 @@ def __post_init__(self):
             self._plugin_configs[plugin_name] = config
 
         if len(user_config.keys()) > 0:
-            raise  # Unprocessed config items
+            raise Exception("Unprocessed config items")
```

Comment:
I like this, can you do this across the board really quick?

Should do better exceptions once #38 is finished